### PR TITLE
Enabled linters checking for all roles and syntax check for examples

### PR DESCRIPTION
**Pull request by KKoukiou**
Also fix all yamllint violations

### CODE REVIEW

**Comment by KKoukiou** [comment]
I think we should run lint and syntax checks for all roles. Will make it much more easy to maintain them.

File: roles/oVirt.clusters/vars/main.yml [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
-     ballooning: true
-     ksm: true
-     host_reason: false
-     vm_reason: false
-     memory_policy: server
-     migration_policy: post_copy
+    description: Development cluster
+    ballooning: true
+    ksm: true
+    ost_reason: false
```
Comment (machacekondra) [inline roles/oVirt.clusters/vars/main.yml]: host_reason

Comment (KKoukiou) [reply roles/oVirt.clusters/vars/main.yml]: oh sorry, fixed.

File: tox.ini [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
 
-commands =
-    yamllint: yamllint examples/
+[testenv:yamllint]
+whitelist_externals =
+    bash
+commands = bash -c "\
+        yamllint \
+                -c {toxinidir}/.yamllint \
+                {toxinidir}/roles"
```
Comment (machacekondra) [inline tox.ini]: I think we should check `tasks/` dir and not roles, no?

Comment (machacekondra) [reply tox.ini]: Ah, sorry, it's infra role :)

Comment (machacekondra) [reply tox.ini]: But still maybe we can check whole root...

Comment (machacekondra) [reply tox.ini]: Also what's the reason of using `bash -c' ?

Comment (KKoukiou) [reply tox.ini]: yamllint will not check the improrted roles from there if I put tasks there, doens't care about import_role.

I can put both.

Comment (machacekondra) [reply tox.ini]: I think the whole root, could be better?

Comment (KKoukiou) [reply tox.ini]: Sure, I 'll need to exclude some files like .travis.yml only.

**Review by machacekondra — APPROVED** [review]

**Pull request merged**